solidity/dashboard/src/contracts.js

```diff
@@ -235,3 +240,17 @@ export function getBondedECDSAKeepFactoryAddress() {
 export function getTBTCSystemAddress() {
   return getContractAddress(TBTCSystem)
 }
+
+const getOldTokenStakingArtifact = async () => {
+  if (
+    process.env.NODE_ENV === "development" ||
+    getFirstNetworkIdFromArtifact() === "3"
```

Comment:
Let's add just a simple clarification comment.
```suggestion
    getFirstNetworkIdFromArtifact() === "3" // ropsten
```

Reply:
pls see 5b9453d